cli-feed-v4.json

```diff
@@ -41,12 +41,12 @@
       "hidden": false
     },
     "v4": {
-      "release": "4.46.0",
+      "release": "4.47.0",
```

Comment:
I think the process here is to update prerelease, make sure it's working by validating with the VS team; then make a new PR to update GA

Reply:
For this particular setting, Fabio mentioned that VS team doesn't require this setting (or get impacted by it). So, we don't need to ask them for validation. Though good to know about the process for pre-release vs release.